ivy/__init__.py

```diff
@@ -1,6 +1,8 @@
 # global
 from types import SimpleNamespace
 import warnings
+
+import ivy
```

Comment:
I think this import may be getting used in one of the previous commits and is unused currently, so we could remove it

Reply:
Yeah not sure why this is there. Might have added it before for some reason 😅